src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -43,28 +41,26 @@ public ResponseEntity endpoint() {
     return ResponseEntity.ok().body(userService.getCurrentUser());
   }
 
-  @JsonView(UserView.UserShort.class)
   @PostMapping("/user")
-  public ResponseEntity<?> createUser(@Valid @RequestBody UserRxDto user) {
+  public ResponseEntity<?> createUser(@RequestBody User user) {
     return ResponseEntity.status(201).body(userService.createUser(user));
   }
 
-  @JsonView(UserView.UserProfile.class)
+
   @GetMapping("/{id}")
   public ResponseEntity<?> readUser(@PathVariable("id") long id) {
     return Optional.of(ResponseEntity.ok().body(userService.readUser(id)))
             .orElse(ResponseEntity.notFound().build());
   }
 
-  @JsonView(UserView.UserBaseId.class)
   @PostMapping("/find")
   public ResponseEntity findAllUsers(@Valid @RequestBody SearchValue str) {
     return Optional.of(ResponseEntity.ok().body(userService.findUsersBySearch(str.getSearch())))
             .orElse(ResponseEntity.notFound().build());
   }
 
   @PutMapping
-  public ResponseEntity<?> updateUser(@Valid @RequestBody UserRxDto user) {
+  public ResponseEntity<?> updateUser(@Valid @RequestBody User user) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_aqAJjEPbGfqJYz&open=AWWU5_aqAJjEPbGfqJYz)

[ISSUE_KEY]: AWWU5_aqAJjEPbGfqJYz